server/src/main/java/org/elasticsearch/indices/SystemIndices.java

```diff
@@ -270,75 +281,22 @@ public ExecutorSelector getExecutorSelector() {
      * @throws IllegalStateException if multiple descriptors match the name
```

Comment:
Nitpick: Can you remove the `@throws` since it no longer throws under any circumstance?